### PR TITLE
Bake in API authorization into v1.1.0 documentation

### LINKED ISSUE #321: Bake in API authorization into v1.1.0 documentation ✨
Using the contents of @coopernetes gist, bake in a manual authorization process that developers can use to complete the end-to-end developer happy path.

### PR DESCRIPTION
**Pull request by JamieSlome**
Closes #321

(cc) @coopernetes

### CODE REVIEW

File: website/docs/get-started/authorize.mdx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+#### 3. Retrieve push with `ID` from database
+
+Using the [cookie](/docs/get-started/authorize#2-authenticate-with-the-api) generated, check that your push with `ID` exists in the database:
+
+```bash
+curl -b ./git-proxy-cookie http://localhost:8080/api/v1/push/${ID}
+```
+
+You should receive a `JSON` blob in the response with details about your push.
```
Comment (coopernetes) [inline website/docs/get-started/authorize.mdx:50]:
````suggestion
Using the [cookie](/docs/get-started/authorize#2-authenticate-with-the-api) generated, execute a GET against the API to confirm that your push with `ID` exists in the database:

```bash
curl -I -b ./git-proxy-cookie http://localhost:8080/api/v1/push/${ID}
```

You should receive a 200 OK in the response.
````

**Comment by JamieSlome** [comment]
@coopernetes @maoo - all discussed changes in our session yesterday have been made. Anything missing?

**Comment by JamieSlome** [comment]
@maoo - the only thing is I want to include the small improvement to the logging in the terminal. It currently has no spacing which makes it very difficult to spot the proxy URL on success.

**Comment by JamieSlome** [comment]
@maoo - that is everything now 👍 Good to merge and then we can release `v1.1.0` 🎉

**Comment by maoo** [comment]
LGTM! I'll leave the honor of merging to you or @coopernetes 😉

**Pull request merged**